lib/lita/handlers/cmd.rb

```diff
@@ -77,13 +81,17 @@ def get_script_list(resp, config)
         bot = Lita::Robot.new
         auth = bot.auth
 
-        list = Dir.entries(config.scripts_dir).select { |f| File.file? "#{config.scripts_dir}/#{f}" }
+        list = Dir.entries(config.scripts_dir).select do |f|
+          (File.file? "#{config.scripts_dir}/#{f}") &&
+          (f.match(config.ignore_scripts))
```

Comment:
Yes - See below, the ignore-script bits were inadvertently included before testing. Feel free to cull them out, or tweak to fix - I think you spotted the issues. I believe I fixed them in my prod environment, but didn't realize I had pushed them in a partial state to github.